src/schema/preparation/source-validation-modules/sidecar-schema.ts

```diff
@@ -125,6 +135,77 @@ const schemaJSON = JSON.parse(`{
           }
         }
       }
+    },
+    "TranslationNamespace": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "types": {
+          "patternProperties": {
+            "^[a-zA-Z0-9_$]+$": {
+              "$ref": "#/definitions/TypeTranslation"
+            }
+          }
+        },
+        "fields": {
+          "patternProperties": {
+            "^[a-zA-Z0-9_$]+$": {
+              "anyOf": [
+                { "$ref": "#/definitions/FieldTranslation" },
+                { "type": "string" }
+              ]
+            }
+          }
+        },
+        "namespaces": {
+          "^[a-zA-Z0-9_$]+$": {
+            "anyOf": [
+              { "$ref": "#/definitions/FieldTranslation" },
+              { "type": "string" }
+            ]
+          }
+        }
+      }
+    },
+    "TypeTranslation": {
```

Comment:
I would use the term `TypeLocalization` here to stay consistent.

Reply:
Good point, although consistent would be TypeI18n.